lldb/examples/python/crashlog.py

```diff
@@ -1794,8 +1803,36 @@ def SymbolicateCrashLogs(debugger, command_args, result, is_command):
         result.SetError(str(e))
         return
 
+    # To avoid breaking existing users, we should keep supporting legacy flags
+    # even if we don't use them / advertise them anymore.
+    if options.interactive:
+        options.mode = CrashLogLoadingMode.interactive
+    elif options.batch:
+        options.mode = CrashLogLoadingMode.batch
```

Comment:
You can't do `-i -b` (or vice versa) because they're in a mutually exclusive group.

To answer the first part of the question, we set `options.mode` here because it's the main "source of truth", so if you did `--mode interactive -b` we would technically override the mode to batch.